public/wp-content/plugins/attendance-plugin/attendance.php

```diff
@@ -1,38 +1,267 @@
+	Description: Plugin for displaying Guests Status according to Events
+	Version: 1.0
+	*/
+	add_action('admin_menu','attendance_page');
+	add_action('admin_enqueue_scripts','cc_plugin_scripts');
+	add_action('admin_enqueue_scripts','cc_plugin_styles');
+	add_action('admin_enqueue_scripts','requested_guest_body');
+	add_action('admin_enqueue_scripts','requested_guest_header');
+	add_action('admin_enqueue_scripts','requested_guest_footer');
+	add_action('admin_enqueue_scripts','invited_guest_body');
+	add_action('admin_enqueue_scripts','invited_guest_header');
+	add_action('admin_enqueue_scripts','invited_guest_footer');
+
+	function attendance_page()
+	{
+		add_menu_page('Attendance','Attendance','manage_options','attendance','attendance');
+	}
+
+	function attendance()
+	{
+		require_once('show-guest-details.php');
+	}
+	
+	function cc_plugin_scripts($hook){
 		if($hook != 'toplevel_page_attendance') {
-                return;
-        }
-        wp_enqueue_script('cc-bootstrap4-script', get_template_directory_uri().'/dist/lib/js/bootstrap4.min.js', array('jquery'), '1.0.0', true);
-        wp_enqueue_script('cc-bootstrap-tether','https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js');
-        wp_enqueue_script('cc-fontawesome-icons','https://use.fontawesome.com/ffc2c94a85.js');
-   }
- function cc_plugin_styles($hook) {
- 		if($hook != 'toplevel_page_attendance') {
-                return;
-        } 
-        wp_enqueue_style('cc-bootstrap4-style', get_template_directory_uri().'/dist/lib/css/bootstrap4.min.css');
-        wp_enqueue_style('cc-fonts','https://fonts.googleapis.com/css?family=Oswald|Marcellus+SC|Roboto|Open+Sans');
-        wp_enqueue_style('style', get_template_directory_uri().'/style.css');
-   }
- 
+			return;
+		}
+		wp_enqueue_script('cc-bootstrap4-script', get_template_directory_uri().'/dist/lib/js/bootstrap4.min.js', array('jquery'), '1.0.0', true);
+		wp_enqueue_script('cc-bootstrap-tether','https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js');
+		wp_enqueue_script('cc-fontawesome-icons','https://use.fontawesome.com/ffc2c94a85.js');
+		wp_enqueue_script('main', get_template_directory_uri().'/main.js', array('jquery'), '1.0.0', true);
+		wp_localize_script( 'main', 'PARAMS', array('ajaxurl' => admin_url('admin-ajax.php')) );
+	}
+	
+	function cc_plugin_styles($hook){
+		if($hook != 'toplevel_page_attendance'){
+			return;
+		} 
+		wp_enqueue_style('cc-bootstrap4-style', get_template_directory_uri().'/dist/lib/css/bootstrap4.min.css');
+		wp_enqueue_style('cc-fonts','https://fonts.googleapis.com/css?family=Oswald|Marcellus+SC|Roboto|Open+Sans');
+		wp_enqueue_style('style', get_template_directory_uri().'/style.css');
+	}
+	 
+	function show_guest_tables(){
+		if(isset($_POST['attendance_event_id'])){
+			$event_id=$_POST['attendance_event_id'];
+			$invited_guest_table=invited_guest_body($event_id);
+			$guest_requests_table=requested_guest_body($event_id);
+			echo $invited_guest_table;
```

Comment:
ajax function should always return values. Don't use echo or printf.

Refer wp_send_json_success().